doc/source/cluster/kubernetes/user-guides/tpu.md

```diff
@@ -0,0 +1,69 @@
+(kuberay-tpu)=
+
+# Using TPUs with KubeRay
+
+This document provides tips on TPU usage with KubeRay.
+
+TPUs are available on Google Kubernetes Engine (GKE). To use TPUs with Kubernetes, configure
+both your Kubernetes setup and add additional values to your RayCluster CR configuration.
+Configure TPUs on GKE by referencing the {ref}`kuberay-gke-tpu-cluster-setup`.
+
+## About TPUs
+
+TPUs are custom-designed AI accelerators, which are optimized for training and inference of large AI models. A TPU host is a VM that runs on a physical computer connected to TPU hardware. TPU workloads can run on one or multiple hosts. A TPU Pod slice is a collection of chips all physically colocated and connected by high-speed inter chip interconnects (ICI). Single-host TPU Pod slices contain independent TPU VM hosts and communicate over the Data Center Network (DCN) rather than ICI interconnects. Multi-host TPU Pod slices contain two more interconnected TPU VM hosts. In GKE, multi-host TPU Pod slices run on their own node pools and are scaled atomically by node pools, rather than individual nodes. Both single-host and multi-host TPU Pod slices can be scaled seamlessly to multiple slices, enabling greater parallelism to support larger workloads.
```

Comment:
```suggestion
TPUs are custom-designed AI accelerators, which are optimized for training and inference of large AI models. A TPU host is a VM that runs on a physical computer connected to TPU hardware. TPU workloads can run on one or multiple hosts. A TPU Pod slice is a collection of chips all physically colocated and connected by high-speed inter chip interconnects (ICI). Single-host TPU Pod slices contain independent TPU VM hosts and communicate over the Data Center Network (DCN) rather than ICI interconnects. Multi-host TPU Pod slices contain two more interconnected TPU VM hosts. In GKE, multi-host TPU Pod slices run on their own node pools and Ray scales them atomically by node pools, rather than individual nodes. Ray scales both single-host and multi-host TPU Pod slices seamlessly to multiple slices, enabling greater parallelism to support larger workloads.
```

Reply:
They're scaled on GKE by node pools, not by Ray. I'll change the wording to be the same but replace Ray with "GKE".

Reply:
[424702e](https://github.com/ray-project/ray/pull/45924/commits/424702e2dbabd9ec815527ec218cac4e9bc1296c)